webpack.config.js

```diff
@@ -61,7 +61,6 @@ module.exports = (env, argv) => {
                             loader: "css-loader",
                             options: {
                                 "sourceMap": !isProduction,
-                                "minimize": isProduction
```

Comment:
Do we want to drop the minimization?

Reply:
The minimization feature of the css-loader got dropped in an earlier version. It only was not breaking the chain, it was just ignored. The output in production mode is still minimized, even without this configuration.

Reply:
Ah cool sounds good.